tests/theia_automation_lsp/README.md

```diff
@@ -12,36 +13,65 @@ Automate functional tests running on theia with Selenium Explorer in ZOS, Explor
 
 2) Install virtualenv module
 
+     Windows:
   	<code>C:\Python3_x64\Scripts\pip3.exe install virtualenv</code>
 
+     Linux: 
+        <code>pip3 install virtualenv</code>
+
 3) Clone the repository
 
+      Windows:
   	<code>C:\WORK>git clone https://github.com/eclipse/che-che4z-lsp-for-cobol.git</code>
-  
+      
+      Linux:
+        <code>$HOME/work$ git clone https://github.com/eclipse/che-che4z-lsp-for-cobol.git</code>
+
 4) Create the virtual environment
+         
+      Windows:
 
   	<code>cd C:\WORK\che-che4z-lsp-for-cobol\theia_automation</code>
 
   	<code>C:\Python3_x64\python.exe -m virtualenv venv3_x64</code>
-  
+
+      Linux:
+
+      <code>cd $HOME/work/che-che4z-lsp-for-cobol/theia_automation</code>
```

Comment:
Please, change the path to cd $HOME/work/che-che4z-lsp-for-cobol/**tests**/theia_automation
And the same for the Windows part.

Reply:
Please check commit 1ba60caf4